themes/classic/modules/ps_contactinfo/nav.tpl

```diff
@@ -29,8 +29,8 @@
       {l
         s='Call us: [1]%phone%[/1]'
         sprintf=[
-          '[1]' => '<span>',
-          '[/1]' => '</span>',
+          '[1]' => "<a href=tel:'{$contact_infos['phone']|replace:' ':''}'>",
```

Comment:
```suggestion
          '[1]' => '<a href="tel:{$contact_infos['phone']|replace:' ':''}">',
```

Reply:
Injecing {$any} vars is not possible on smarty, you should use double quotes such as "